components/modal/style.scss

```diff
@@ -0,0 +1,104 @@
+.components-modal__screen-overlay {
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+	bottom: 0;
+	background-color: rgba(0, 0, 0, 0.6);
+	z-index: z-index( '.components-modal__screen-overlay' );
+	// on mobile the main content area has to scroll
+	// otherwise you can invoke the overscroll bounce on the non-scrolling container, causing (ノಠ益ಠ)ノ彡┻━┻
+	@include break-small {
+		position: fixed;
+		top: $admin-bar-height-big;
+	}
+
+	@include break-medium() {
+		top: $admin-bar-height;
+	}
+}
+
+.components-modal__frame {
+	position: absolute;
+	top: 50%;
+	left: 50%;
+	right: auto;
+	bottom: auto;
+	width: auto;
+	max-width: calc( 100vh - 200px );
+	max-height: 90%;
+	border: 0;
+	border-radius: 0;
+	margin-right: -50%;
+	transform: translate(-50%, -50%);
+	background-color: #fff;
+	outline: none;
+	box-shadow: 0 3px 25px #000;
+
+	// In small screens the content needs to be full width.
+	@media ( max-width: #{ ( $break-small ) } ) {
+		position: fixed;
+		top: 0;
+		bottom: 0;
+		right: 0;
+		left: 0;
+		margin: 0;
+		transform: initial;
+		max-width: none;
+		max-height: none;
+	}
+}
+
+.components-modal__header {
+	box-sizing: border-box;
+	height: $header-height;
+	border-bottom: 1px solid $light-gray-500;
+	padding: 10px;
+	display: flex;
+	flex-direction: row;
+	align-items: stretch;
+	justify-content: space-between;
+	z-index: z-index( '.edit-post-header' );
+	left: 0;
+	right: 0;
+
+	div {
+		align-items: center;
+		flex-grow: 1;
+		display: flex;
+		flex-direction: row;
+		justify-content: left;
+
+		h1 {
+			font-size: 1em;
+			font-weight: normal;
+			padding-left: 5px;
+		}
+
+		span {
```

Comment:
Element selectors should be generally discouraged since it's likely to introduce an inadvertent cascade (either by future maintenance or custom-provided children values) when you intend to target only a single item in this instance.